js/utilities/renderChallenges.js

```diff
@@ -0,0 +1,27 @@
+import Challenge from '../challengeClass.js';
+
+export const renderChallenges = (challenges, container, withButtons = true) => {
```

Comment:
That is very smart!
Anything that repeats more than twice should be made into a function :)